resources/views/index.blade.php

```diff
@@ -12,8 +12,13 @@
         <style>{!! file_get_contents($cssPath) !!}</style>
     @endisset
     @livewireStyles
+    @if(session('log-viewer:theme', 'system') == 'system')
+        <script>
+            document.documentElement.classList.add('dark')
```

Comment:
Also, I just realised there's a bug here. I forgot to add `window.matchMedia("prefers-color-scheme...")` thing. 🤣